### ISSUE TITLE
Consulta sobre sugerencia

### ISSUE THREAD

**Issue by angelina1sys** [angelina1sys/Paquetemeteored#3]
Hola profes! Buenas tardes, estamos con ines modificando las sugerencias recibidas sobre nuestro paquete meteored y no logramos entender a qué se refieren con la siguiente sugerencia: "No encontramos un archivo que prepare los datos." ¿Qué debería contener ese archivo que nos falta y donde debe ubicarse?
Esperamos su respuesta, gracias!
@paocorrales @yabellini

**Comment by yabellini** [comment]
En general en la carpeta data_raw se colocan los datos crudos y el script que los limpia y los genera en su forma final en el paquete.

También allí se suele encontrar la función que agrega los datos al paquete. 

Luego esa carpeta se debe agregar a los archivos que se deben ignorar cuando se genera el paquete.

**Comment by angelina1sys** [comment]
Ahí intentamos hacer el archivo para preparar los datos, con ines quisiéramos saber si esta bien hecho
@yabellini

**Comment by paocorrales** [comment]
A simple vista parece correcto. 

Les recomiendo revisar las dependencias de las funciones del paquete, eso es lo que les está dando error en R CMD checks.

**Comment by angelina1sys** [comment]
Hola profes! Nos está tirando error los tests, no entendemos los errores ya que los corregimos pero sigue figurando con esos mismos errores. Nos preocupa no llegar a la nota minima aprobatoria debido a estos inconvenientes. Lo estuvimos viendo con un par de nuestros compañeros y ni siquiera ellos saben qué es lo que sucede. Podremos hacer un meet o zoom para ver como solucionarlo? o de lo contrario, un feedback completo para que podamos solucionarlo con mi compañera ines.
@yabellini @paocorrales

**Comment by paocorrales** [comment]
No hay problema, ahora les mando un mail para coordinar.

**Comment by paocorrales** [comment]
Hola nuevamente! Tuve tiempo de mirar el paquete, en particular lo que devuelven los R CMD checks que corren en github y veo que uno de los tests les da error:

```
══ Failed tests ════════════════════════════════════════════════════════════════
── Error ('test-grafico_temperatura_mensual.R:7:3'): grafico_temperatura_mensual funciona correctamente ──
Error in `eval(code, test_env)`: object 'NH0046' not found
```

No se si es este el problema que no está pudiendo solucionar. Acá dice que no encuentra el objeto NH0046 y por lo que veo en el paquete, no hay ningun set de datos con ese nombre. Tal vez esto les ayuda.

**Comment by angelina1sys** [comment]
Claro! Lo que sucede es que nosotros teníamos ese dataset, luego lo eliminamos para descargarlo con la función descargar datos, y cambiamos el nombre, la función anda la probamos aparte ejecutando sus líneas pero el check nos detecta eso que ya está corregido
